web/src/components/feature/package_dialog/ComponentTableSum.tsx

```diff
@@ -0,0 +1,33 @@
+import { EdsProvider, Table } from '@equinor/eds-core-react'
+import { TComponentRatios } from '../../../types'
+import { formatNumber } from '../../../tableUtils'
+
+function computeComponentRatioSum(componentRatios: TComponentRatios) {
+  if (Object.keys(componentRatios).length > 0) {
+    return formatNumber(
+      Object.values(componentRatios).reduce((a, b) => a + b),
+      3
+    )
+  } else {
+    return 0
+  }
+}
```

Comment:
```suggestion
function computeComponentRatioSum(componentRatios: TComponentRatios) {
  return formatNumber(
    Object.values(componentRatios).reduce((a, b) => a + b, 0),
    3
  )
}
```

Reply:
Huh. It works now, but did not work earlier today when I tried. Got: (react-dom.development.js:12056 Uncaught TypeError: Reduce of empty array with no initial value).

Reply:
That's because you forgot to give reduce it's initial value